crates/subspace-farmer/src/plot.rs

```diff
@@ -344,51 +344,68 @@ impl Plot {
         })?
     }
 
-    /// Returns sequential piece indexes for piece retrieval
-    pub(crate) fn read_sequential_piece_indexes(
-        &self,
-        from_index_hash: PieceIndexHash,
-        count: u64,
-    ) -> io::Result<Vec<PieceIndex>> {
-        let (result_sender, result_receiver) = mpsc::channel();
-
-        self.inner
-            .requests_sender
-            .send(RequestWithPriority {
-                request: Request::ReadPieceIndexes {
-                    from_index_hash,
-                    count,
-                    result_sender,
-                },
-                priority: RequestPriority::Low,
-            })
-            .map_err(|error| {
-                io::Error::other(format!(
-                    "Failed sending read piece indexes request: {error}"
-                ))
-            })?;
-
-        result_receiver.recv().map_err(|error| {
-            io::Error::other(format!(
-                "Read piece indexes result sender was dropped: {error}",
-            ))
-        })?
-    }
+    const PIECES_PER_REQUEST: u64 = 1000;
```

Comment:
It would be ideal to have constants at the top and not in random places, it helps with discoverability